src/authorizations/components/redesigned/TokenRow.test.tsx

```diff
@@ -0,0 +1,57 @@
+// Libraries
+import {fireEvent, waitFor} from '@testing-library/react'
+import React from 'react'
+
+// Fixtures
+import TokenRow from './TokenRow'
+import {auth} from 'mocks/dummyData'
+import {renderWithRedux} from 'src/mockState'
+
+// Actions
+import {deleteAuthorization} from 'src/client'
+
+jest.mock('src/client', () => ({
+  deleteAuthorization: jest.fn(() => ({
+    headers: {},
+    status: 201,
+  })),
+}))
+
+const setup = (override?) => {
+  const authorization = {
+    ...auth,
+    description: 'XYZ',
+    createdAt: '2020-08-19T23:13:44.514Z',
+    updatedAt: '2020-08-19T23:13:44.514Z',
+  }
+
+  const props = {
+    auth: authorization,
+    onClickDescription: jest.fn(),
+    onClone: jest.fn(),
+    ...override,
+  }
+  return renderWithRedux(<TokenRow {...props} />)
+}
+
+describe('TokenRowResourceCard', () => {
+  it('should render', () => {
+    const {getByTestId} = setup()
+    expect(getByTestId('token-card XYZ')).toBeDefined()
+    expect(getByTestId('token-name XYZ')).toBeDefined()
+  })
+
+  it('displays delete button', () => {
+    const {getByTestId} = setup()
+
+    expect(getByTestId('delete-token')).toBeVisible()
+  })
+
+  it('executes the delete method', async () => {
+    const {getByTestId} = setup()
+
+    fireEvent.click(getByTestId('delete-token'))
+    await waitFor(() => expect(deleteAuthorization).toBeCalled())
+    expect(deleteAuthorization).toBeCalledTimes(1)
```

Comment:
right now, this test is a bit tautological. E.g. "call the delete method, was it called? yep it was called."

I think we can make it a little more valuable by asserting that the row we deleted is no longer there. I'm not sure if that's even possible without changing the store to 'simulate' a delete event, so this comment might be a little more theoretical in nature.

When we test things, we want to test the effect of the operation on the system. In this case, we want to test that when we click the delete button, we call the delete api method (:+1: on that) *and* that the deleted item no longer shows up in the data and the UI. Since this is a mocked, that last step is kind of difficult when we mock out the API layer. The general flow is
1. click delete
1. call API to delete stuff
1. API does API stuff
1. call API to get updated list of items
1. render new list without that deleted item

to do that without an actual API, we'd have to mock out steps 2 and 4 (3 is simulated). This test currently mocks out step 2. To really test this, we'd have to also mock out test 4, but that's still kind of fake.

Hope that explanation was useful and not a wall of obvious text

Reply:
like we talked about on a Zoom call, this case is covered by an e2e test, so we should just remove this